### PR TITLE
Register admin

### PR DESCRIPTION
**Pull request by Egoes98**
Added register admin window on the admin panel. Also avery time you run the server it creates a default admin because you need an admin to cerate more.

### CODE REVIEW

**Review by iapellaniz — COMMENTED** [review]
We should remove phone, as it's not stored for Administrators

**Comment by Egoes98** [comment]
> We should remove phone, as it's not stored for Administrators

Its alredy changed.

File: src/main/java/es/deusto/spq/server/remote/IHotelManager.java [hunk 1]
```diff
@@ -13,7 +13,17 @@
 public interface IHotelManager extends Remote {
 
 	public UserDTO signInGuest(String name, String email, String password, String phone, String address) throws RemoteException;
-	//TODO signInAdministrator
+	/**
+	 * The method that creates the admin
+	 * @param name
+	 * @param email
+	 * @param password
+	 * @param phone
```
Comment (iapellaniz) [inline src/main/java/es/deusto/spq/server/remote/IHotelManager.java]: Remove phone mentions from code

File: src/main/resources/localization_en_US.properties [hunk 2]
```diff
@@ -37,6 +37,23 @@ register.validation.title = Form submission problem
 register.success.body = Your information has been saved. You can now log in.
 register.success.title = Success
 
+# Register admin Window
+register.admin.title = Register:
+register.admin.label.name = Name:
+register.admin.label.email = E-mail:
+register.admin.label.password = Password:
+register.admin.label.password-confirmation = Password Confirmation:
+register.admin.label.phone = Phone number:
```
Comment (iapellaniz) [inline src/main/resources/localization_en_US.properties]: Remove phone mentions from code

File: src/main/java/es/deusto/spq/client/gui/base/ViewFactory.java [hunk 3]
```diff
@@ -42,8 +43,10 @@ public static View buildView(ViewType type, ViewManager viewManager) {
             case GUEST_HOTELS:
             	view = new HotelGuestView(viewManager);
             	break;
+            case REGISTER_ADMINISTRATOR:
```
Comment (iapellaniz) [inline src/main/java/es/deusto/spq/client/gui/base/ViewFactory.java:47]: New line before `case` please

**Review by iapellaniz — CHANGES_REQUESTED** [review]

**Review by iapellaniz — APPROVED** [review]

**Pull request merged**